README.rst

```diff
@@ -37,21 +40,27 @@ PyVista
 .. |gitter| image:: https://img.shields.io/gitter/room/pyvista/community?color=darkviolet
    :target: https://gitter.im/pyvista/community
 
-
-
-+----------------------+------------------------+
-| Deployment           | |pypi| |conda|         |
-+----------------------+------------------------+
-| Build Status         | |azure|                |
-+----------------------+------------------------+
-| Metrics              | |codacy| |codecov|     |
-+----------------------+------------------------+
-| Citation             | |joss| |zenodo|        |
-+----------------------+------------------------+
-| License              | |MIT|                  |
-+----------------------+------------------------+
-| Community            | |slack| |gitter|       |
-+----------------------+------------------------+
+.. |PyPIact| image:: https://img.shields.io/pypi/dm/pyvista.svg?label=PyPI%20downloads
+.. _PyPI: https://pypi.org/project/pyvista/
+
+.. |condaact| image:: https://img.shields.io/conda/dn/conda-forge/pyvista.svg?label=Conda%20downloads
+.. _conda-forge: https://anaconda.org/conda-forge/pyvista
```

Comment:
Are the PyPI and conda-forge links unused? Were they meant to be `:target:`s instead?

Reply:
Fixed